internal/pkg/cli/pipeline_init.go

```diff
@@ -7,21 +7,26 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/manifest"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store/secretsmanager"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store/ssm"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/prompt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
 
 	"github.com/spf13/cobra"
 )
 
 const (
 	pipelineAddEnvPrompt          = "Would you like to add an environment to your pipeline?"
 	pipelineSelectEnvPrompt       = "Which environment would you like to add to your pipeline?"
-	pipelineEnterGitHubRepoPrompt = "What is your application's GitHub repository?"
+	pipelineEnterGitHubRepoPrompt = "What is your application's GitHub repository? (Enter full repository URL, e.g. https://github.com/myCompany/myRepo)" // TODO allow just <user>/<repo>?
```

Comment:
What do you think of moving the example to the help text of the prompt?
and the prompt itself can be "What is your application's GitHub repository URL?"